gui/hud-gui.lua

```diff
@@ -641,7 +653,7 @@ function gui_hud.calculate_hud_size(player_index)
 		common.debug_log(player_index, "Start calculating HUD size:")
 
 		local max_columns_allowed = player_settings.get_hud_columns_setting(player_index)
-		width = (max_columns_allowed * (36 + 4)) + 40
+		width = (max_columns_allowed * (52 + 4)) + 40
```

Comment:
The "52" here comes from measuring the signal number width in an image editor.

Reply:
Yes, and base the number on the toggled setting as not to break the current layout